x/recovery/spec/02_hooks.md

```diff
@@ -1,5 +1,8 @@
 <!--
-order: 3
+order: 2
+title: "Recovery hooks"
+parent:
+  title: "recovery"
```

Comment:
```suggestion
order: 2
```

Remove the custom titles and parents on each sub page, as you are actually changing the title in the navigation to `Recovery hook` with this and it should simply be `Hooks`. You can just level it out. Also you can build the docs with `make docs-serve` to test your changes